gctrpc/README.md

```diff
@@ -37,20 +37,29 @@ Then use `go get` to download the following packages:
 
 ```bash
 go get github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+go get github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
 go get google.golang.org/protobuf/cmd/protoc-gen-go
 go get google.golang.org/grpc/cmd/protoc-gen-go-grpc
 ```
 
-This will place three binaries in your `$GOBIN`;
+This will place the following binaries in your `$GOBIN`;
 
 * `protoc-gen-grpc-gateway`
-* `protoc-gen-swagger`
+* `protoc-gen-openapiv2`
 * `protoc-gen-go`
 * `protoc-gen-go-grpc`
 
 Make sure that your `$GOBIN` is in your `$PATH`.
 
+There was a name change we need to address: `protoc-gen-swagger` is
```

Comment:
I think instead of this, we can just update the build script to switch from `--swagger_out=` to `--openapiv2_out`. So the whole path will become:

`
protoc -I=. -I=%GOPATH%\src -I=%GOPATH%\src\github.com\grpc-ecosystem\grpc-gateway\third_party\googleapis --openapiv2_out=logtostderr=true:. rpc.proto
`